cli/api.py

```diff
@@ -112,6 +113,18 @@ def wrapped(*args, **kwargs):
     return wrapped
 
 
+class RequestsWithReauth:
```

Comment:
if it's a service (private?) class, wouldn't it be right to name it starting from underscore or something?